<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportZNOEstablishedFirst</class>
 <widget class="QDialog" name="ReportZNOEstablishedFirst">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="1">
    <widget class="QComboBox" name="cmbEmployment">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Да</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Нет</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbZNO">
     <property name="title">
      <string>ЗНО</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbZNOFirst">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbZNOMorph"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblZNOFirst">
        <property name="text">
         <string>ЗНО установлен впервые</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblZNOMorph">
        <property name="text">
         <string>ЗНО подтверждён морфологически</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="chkEmployment">
     <property name="text">
      <string>Трудоспособность</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="chkSex">
     <property name="text">
      <string>Пол</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="CRBListBox" name="lstOrgStructure"/>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="CRBListBox" name="lstEventType"/>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="chkMKB">
     <property name="text">
      <string>Коды диагнозов по МКБ</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbDistrict"/>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="chkOrgStructureMulti">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="CICDCodeEdit" name="edtMKBFrom">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>a00.00; </string>
       </property>
       <property name="text">
        <string>A.</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CICDCodeEdit" name="edtMKBTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>a00.00; </string>
       </property>
       <property name="text">
        <string>Z99.9</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="cmbSex">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Не указано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QCheckBox" name="chkDistrictMulti">
     <property name="text">
      <string>Район</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkEventTypeMulti">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="3">
    <widget class="CRBListBox" name="lstDistrict"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="chkAge">
     <property name="text">
      <string>Возраст</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="edtAgeFrom">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>160</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="edtAgeTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QCheckBox" name="chkGroupOrgStructure">
     <property name="text">
      <string>Группировать по подразделениям</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinanceType"/>
   </item>
   <item row="16" column="0">
    <widget class="QCheckBox" name="chkFinanceTypeMulti">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="3">
    <widget class="CRBListBox" name="lstFinanceType"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CICDCodeEdit</class>
   <extends>QLineEdit</extends>
   <header>library/ICDCodeEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QListView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportZNOEstablishedFirst</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportZNOEstablishedFirst</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMKB</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtMKBTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkMKB</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtMKBFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSex</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbSex</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAge</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtAgeTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAge</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtAgeFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEmployment</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbEmployment</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEventTypeMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbEventType</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEventTypeMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstEventType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrgStructureMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbOrgStructure</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOrgStructureMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstOrgStructure</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDistrictMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbDistrict</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDistrictMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstDistrict</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>715</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFinanceTypeMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbFinanceType</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFinanceTypeMulti</sender>
   <signal>toggled(bool)</signal>
   <receiver>lstFinanceType</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>663</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
